src/tasks/BagelAITools.py

```
@@ -28,105 +36,693 @@ class BagelAITools(NTEOneTimeTask, BaseNTETask):
             {
+                self.CONF_GAME_LANG : "游戏语言/遊戲語言/ゲーム言語/게임 언어\nGame Language/Язык игры",
                 self.CONF_MODEL: "关闭后将降级使用本地词库抽取发帖回复文案",
                 self.CONF_HELPER_MODE: "开启助手模式后，将只会辅助生成文案",
                 self.CONF_AUTO_AICONFIG: "智能体模式选项\n自动回帖会同时点赞",
                 self.CONF_MODEL_URL: "使用模型根据图片生成文案，推荐本地部署",
                 self.CONF_MODEL_API: "未设置请留空，请勿泄露API_Key！",
                 self.CONF_MODEL_NAME: "推荐qwen/qwen3-vl-4b，显存占用较小",
-                self.CONF_PROMPT_REPLY: "回复模块提示词，请先调试好文案再使用",
-                self.CONF_PROMPT_POST_TITLE: "发帖标题模块提示词，请先调试好文案再使用",
-                self.CONF_PROMPT_POST_CONTENT: "发帖内容模块提示词，请先调试好文案再使用",
+                self.CONF_PROMPT_REPLY: f"回复模块提示词，请先调试好文案再使用，示例：\n{self.model_prompt["REPLY"]}",
```

Ruff (0.15.15)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)